src/main/java/org/sharedid/endpoint/config/RouteConfiguration.java

```diff
@@ -64,8 +64,9 @@ public Router router(Vertx vertx, HandlerRegistry handlerRegistry, TemplateEngin
                                 .allowedMethod(HttpMethod.POST)
                                 .allowedMethod(HttpMethod.OPTIONS));
 
-        router.get("/")
-                .handler(StaticHandler.create().setIndexPage("index.html"));
+        router.get("/lib/*").handler(StaticHandler.create("webroot/lib"));
+        router.get("/keys/*").handler(StaticHandler.create("webroot/keys"));
```

Comment:
this handler suggests you are trying to serve the keys from the id application. However, it is hosted at id.sharedid.org not sharedId.org, if so URL_SHARED_ID_PUBLIC_KEY above is likely the wrong domain.

Reply:
for example: https://sharedid.org/usync is not a valid url, but https://id.sharedid.org/usync is.

Reply:
will update it. https://sharedid.org/index.html exists, so I assumed it's deployed there